migration_original/ODS1Stage/tables/Base/ProviderLicense/spu_translated_ProviderLicense.sql

```diff
@@ -0,0 +1,124 @@
+CREATE OR REPLACE PROCEDURE ODS1_STAGE.BASE.SP_LOAD_PROVIDERLICENSE()
+    RETURNS STRING
+    LANGUAGE SQL
+    EXECUTE AS CALLER
+    AS  
+DECLARE 
+---------------------------------------------------------
+--------------- 0. Table dependencies -------------------
+---------------------------------------------------------
+    
+-- Base.ProviderLicense depends on: 
+--- Raw.PROVIDER_PROFILE_PROCESSING
+--- Base.Provider
+--- Base.ProviderMalpractice
+
+---------------------------------------------------------
+--------------- 1. Declaring variables ------------------
+---------------------------------------------------------
+
+    select_statement STRING; -- CTE and Select statement for the Merge
+    delete_statement STRING; -- Delete statement 
+    insert_statement STRING; -- Insert statement for the Merge
+    merge_statement STRING; -- Merge statement to final table
+    status STRING; -- Status monitoring
+   
+---------------------------------------------------------
+--------------- 2.Conditionals if any -------------------
+---------------------------------------------------------   
+   
+BEGIN
+    -- no conditionals
+
+
+---------------------------------------------------------
+----------------- 3. SQL Statements ---------------------
+---------------------------------------------------------     
+
+--- Select Statement
+select_statement := $$ SELECT DISTINCT
+                            P.ProviderID,
+                            JSON.License_State AS StateID, 
```

Comment:
Using state explicitly as an id. I think we should change this.

Reply:
Done